frankenphp.c

```diff
@@ -405,7 +405,15 @@ static size_t frankenphp_ub_write(const char *str, size_t str_length)
 		return 0;
 	}
 
-	return go_ub_write(ctx->current_request ? ctx->current_request : ctx->main_request, (char *) str, str_length);
+	bool failed = false;
+
+	size_t wrote = go_ub_write(ctx->current_request ? ctx->current_request : ctx->main_request, (char *) str, str_length, &failed);
+
+	if(failed) {
```

Comment:
```suggestion
	if (failed) {
```

(we need to check and fix the CS in the CI btw)

Reply:
Is there a C-style checker? I'll admit I only recently noticed the space between if & (.